modules/commons/src/main/resources/contexts/neurosciencegraph/core/data/v1.0.3.json

```diff
@@ -0,0 +1,594 @@
+      },
+      "thirdRow": {
+        "@id": "nsg:thirdRow"
+      },
+      "voxelResolution": {
+        "@id": "nsg:voxelResolution"
+      },
+      "voxelResolutionValue": {
+        "@id": "nsg:voxelResolutionValue"
+      },
+      "boundingbox": {
+        "@id": "nsg:boundingbox"
+      },
+      "isRegisteredIn": {
+        "@id": "nsg:isRegisteredIn"
+      },
+      "sliceDirection": {
+        "@id": "nsg:sliceDirection"
+      },
+      "sliceWidth": {
+        "@id": "nsg:sliceWidth"
+      },
+      "sliceHeight": {
+        "@id": "nsg:sliceHeight"
+      },
+      "numberOfSlices": {
+        "@id": "nsg:numberOfSlices"
+      },
+      "sliceResolution": {
+        "@id": "nsg:sliceResolution"
+      },
+      "sliceInterval": {
+        "@id": "nsg:sliceInterval"
+      },
+      "widthResolution": {
+        "@id": "nsg:widthResolution"
+      },
+      "heightResolution": {
+        "@id": "nsg:heightResolution"
+      },
+      "sliceIntervalValue": {
+        "@id": "nsg:sliceIntervalValue"
+      },
+      "emodelIndex": {
+        "@id": "nsg:emodelIndex"
+      },
+      "morphologyIndex": {
+        "@id": "nsg:morphologyIndex"
+      },
+      "eModel": {
+        "@id": "nsg:eModel"
+      },
+      "emodelRelease": {
+        "@id": "nsg:emodelRelease"
+      },
+      "morphologyRelease": {
+        "@id": "nsg:morphologyRelease"
+      },
+      "memodelIndex": {
+        "@id": "nsg:memodelIndex"
+      },
+      "circuitCellProperties": {
+        "@id": "nsg:circuitCellProperties"
+      },
+      "property": {
+        "@id": "nsg:property"
+      },
+      "edgeCollection": {
+        "@id": "nsg:edgeCollection"
+      },
+      "morphology": {
+        "@id": "nsg:morphology"
+      },
+      "mainModelScript": {
+        "@id": "nsg:mainModelScript"
+      },
+      "name": {
+        "@id": "schema:name"
+      },
+      "version": {
+        "@id": "schema:version"
+      },
+      "isPartOf": {
+        "@id": "dcterms:isPartOf",
+        "@type": "@id"
+      },
+      "view2d": {
+        "@id": "nsg:view2d"
+      },
+      "view3d": {
+        "@id": "nsg:view3d"
+      },
+      "wasDerivedFrom": {
+        "@id": "prov:wasDerivedFrom"
+      },
+      "experimentalCellList": {
+        "@id": "nsg:experimentalCellList"
+      },
+      "masterListConfiguration": {
+        "@id": "nsg:masterListConfiguration"
+      },
+      "experimentalTraceLocation": {
+        "@id": "nsg:experimentalTraceLocation"
+      },
+      "stimuliToExperimentMap": {
+        "@id": "nsg:stimuliToExperimentMap"
+      },
+      "featureExtractionConfiguration": {
+        "@id": "nsg:featureExtractionConfiguration"
+      },
+      "configurationUsed": {
+        "@id": "nsg:configurationUsed"
+      },
+      "features": {
+        "@id": "nsg:features"
+      },
+      "hypampThreshold": {
+      "@id": "nsg:hypampThreshold"
+      },
+      "storageType": {
+        "@id": "nsg:storageType"
```

Comment:
I suggest you add "@type" :"@id" so that the context gets applied for this term.

Reply:
Done